passthrough.go

```diff
@@ -49,7 +49,7 @@ Use a version name to specify a named version or a height to execute on the vers
 					return fmt.Errorf("version %s not found: %w", versionName, err)
```

Comment:
in a follow-up (or here) we should change name by app version, this will allow us to remove name.